cfgov/v1/forms.py

```diff
@@ -116,6 +116,14 @@ def clean(self, value):
                 pass
         return value
 
+class PDFFilterDateField(forms.DateField):
```

Comment:
@sebworks didn't have time to look for it today, but could you push up a change to this branch?